code/game/objects/items/devices/violin.dm

```diff
@@ -1,392 +1,41 @@
 //copy pasta of the space piano, don't hurt me -Pete
-
-/obj/item/device/violin
-	name = "space violin"
-	desc = "A wooden musical instrument with four strings and a bow. \"The devil went down to space, he was looking for an assistant to grief.\""
+/obj/item/device/instrument

```

Comment:
I hestiated to just rename this to instrument.dm, but should any hand-held instruments be added, it may be prudent to rename it then.